src/Morphology/Morphology.jl

```diff
@@ -1,18 +1,48 @@
 module Morphology
 
+export imregionalmin, imextendedmin, bwdist
+
+import Images.ImageMorphology: local_minima, distance_transform, feature_transform
+import Images.ImageSegmentation: hmin_transform
+
+"""
+    imregionalmin(img, conn=2)
+
+Compute the regional minima of the image `img` using the connectivity `conn`.
+
+Returns a bitmatrix of the same size as `img` with the regional minima.
+
+# Arguments
+- `img`: Image object
+- `conn`: Neighborhood connectivity; in 2D, 1 = 4-neighborhood and 2 = 8-neighborhood
 """
-    dummy_morphology_function()
+function imregionalmin(img, conn=2)
+    return local_minima(img; connectivity=conn) .> 0
+end
 
-Example function for morphology module
+"""
+    imextendedmin(img)
 
-!!! todo "Delete this once real functions are added"
-    This function should be removed when real morphology functions are moved into this module.
+Mimics MATLAB's imextendedmin function that computes the extended-minima transform, which is the regional minima of the H-minima transform. Regional minima are connected components of pixels with a constant intensity value. This function returns a transformed bitmatrix.
 
+# Arguments
+- `img`: image object
+- `h`: suppress minima below this depth threshold
+- `conn`: neighborhood connectivity; in 2D 1 = 4-neighborhood and 2 = 8-neighborhood
 """
-function dummy_morphology_function()
-    return "This is a dummy morphology function."
+function imextendedmin(img::AbstractArray, h::Int=2, conn::Int=2)::BitMatrix
+    mask = hmin_transform(img, h)
+    mask_minima = local_minima(mask; connectivity=conn)
+    return mask_minima .> 0
```

Comment:
These two lines can be replaced with a call to `imregionalmin`.

```suggestion
    mask_minima = imregionalmin(mask; connectivity=conn)
    return mask_minima
```

Reply:
As a counterpoint, I think it would be more straightforward to keep using the ImageMorphology function `local_minima` rather than introduce a wrapper that essentially only renames the function to match the Matlab function. If we were extending the `local_minima` to have deeper functionality then I think it makes sense to have another function. If we really need only one line you could replace

```julia
mask_minima = local_minima(mask; connectivity=conn)
return mask_minima .> 0
```

with

```julia
return local_minima(mask; connectivity=conn) .> 0
```
(or use convert(BitMatrix, ) if you want).